### ISSUE TITLE
Failed to resolve: com.estimote:scanning-sdk:0.1.0

### ISSUE THREAD

**Issue by Zhuinden** [Estimote/Android-SDK#257]
Added 

      compile 'com.estimote:sdk:1.3.0'

fails as dependency is not found.

**Comment by Zhuinden** [comment]
compile 'com.estimote:sdk:1.2.0'

seems to work

**Comment by wafel82** [comment]
Hi @Zhuinden 

Thank you for reporting this. Confirmed - it's our bug.  By mistake one of our new SDK components (scanning-sdk) was released internally only, so we (Estimoters) were able to use it without any issues however it wan not accessible publicly - our bad :( . 
**I've just requested to add scanning-sdk component to jCenter repo so the issue should be resolved in several hours (as soon as the Bintray process repos population)**
 
We're in the middle of huge architectural refactoring of our SDK, which - we hope - will bring our SDKs quality and overall experience on whole new level. Stay tuned - new releases fully build up on new architecture will be available in upcoming weeks.
In the mean time there might be some glitches in the build process - we're apologise for that.

Regards
Wojtek
